### PR TITLE
Add personal detail page for app members

### PR DESCRIPTION
**Pull request by jessieay**
* Page asks for sex, marital status, SSN
* Introducing concept of "members" since we will soon be asking this
  for multiple people.

screen shot 2017-08-10 at 6 50 02 pm

### CODE REVIEW

**Comment by jessieay** [comment]
[As an MDHHS worker, I want to know more details about the person applying so I can identify them in my system.](https://trello.com/c/eP065iH9/96-as-an-mdhhs-worker-i-want-to-know-more-details-about-the-person-applying-so-i-can-identify-them-in-my-system)

**Comment by jayroh** [comment]
We should consider bumping up the inclusion and setup of the `attr_encrypted` gem so items like SSN get encrypted on disk. Just a note to get that on our radar before we start taking on real applicants.

File: app/controllers/introduce_yourself_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class IntroduceYourselfController < StepsController
+  def edit
+    current_app = current_or_new_snap_application
+
+    @step = step_class.new(
+      HashWithIndifferentAccess.
+        new(current_app.attributes).
+        slice(*step_attrs),
+    )
```
Comment (jayroh) [inline app/controllers/introduce_yourself_controller.rb]: what do you think about extracting `HashWithIndifferentAccess.new(current_app.attributes).slice(*step_attrs)` out to something like?

```ruby
def existing_attributes  # or, like below, `snap_application_attributes`
  HashWithIndifferentAccess.
    new(current_app.attributes).
    slice(*step_attrs)
end
```

File: app/helpers/mb_form_builder.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 class MbFormBuilder < ActionView::Helpers::FormBuilder
-  def mb_input_field(method, label_text, type: "text", notes: [], options: {}, classes: [], prefix: nil, autofocus: nil)
+  def mb_input_field(method, label_text, type: "text", notes: [], options: {}, classes: [], prefix: nil, autofocus: nil, append_html: "")
     classes = classes.append(%w[text-input])
     <<-HTML.html_safe
       <fieldset class="form-group#{error_state(object, method)}">
         #{label_and_field(method, label_text, text_field(method, { autofocus: autofocus, type: type, class: classes.join(' '), autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', spellcheck: 'false' }.merge(options)), notes: notes, prefix: prefix)}
         #{errors_for(object, method)}
+        #{append_html}
```
Comment (jayroh) [inline app/helpers/mb_form_builder.rb]: this is handy!

File: app/controllers/introduce_yourself_controller.rb (hunk 1 above)
Comment (jessieay) [inline app/controllers/introduce_yourself_controller.rb]: ok I will do this in a follow up so we can have Alan accept today :)

**Pull request merged**